src/equations/linearized_euler_2d.jl

```diff
@@ -143,6 +161,187 @@ end
     end
 end
 
+@inline function max_abs_speed_naive(u_ll, u_rr, normal_direction::AbstractVector,
+                                     equations::LinearizedEulerEquations2D)
+    @unpack v_mean_global, c_mean_global = equations
+    v_mean_normal = normal_direction[1] * v_mean_global[1] +
+                    normal_direction[2] * v_mean_global[2]
+    return abs(v_mean_normal) + c_mean_global * norm(normal_direction)
+end
+
+@inline function flux_upwind(u_ll, u_rr, orientation::Integer,
+                             equations::LinearizedEulerEquations2D)
+    @unpack v_mean_global, rho_mean_global, c_mean_global = equations
+    v1_mean = v_mean_global[1]
+    v2_mean = v_mean_global[2]
+
+    rho_prime_ll, v1_prime_ll, v2_prime_ll, p_prime_ll = u_ll
+    rho_prime_rr, v1_prime_rr, v2_prime_rr, p_prime_rr = u_rr
+
+    if orientation == 1
+        # Eigenvalues of the flux matrix
+        lambda1 = v1_mean
+        lambda2 = v1_mean - c_mean_global
+        lambda3 = v1_mean + c_mean_global
+
+        lambda1_p = positive_part(lambda1)
+        lambda2_p = positive_part(lambda2)
+        lambda3_p = positive_part(lambda3)
+        lambda2p3_half_p = 0.5 * (lambda2_p + lambda3_p)
+        lambda3m2_half_p = 0.5 * (lambda3_p - lambda2_p)
+
+        lambda1_m = negative_part(lambda1)
+        lambda2_m = negative_part(lambda2)
+        lambda3_m = negative_part(lambda3)
+        lambda2p3_half_m = 0.5 * (lambda2_m + lambda3_m)
+        lambda3m2_half_m = 0.5 * (lambda3_m - lambda2_m)
+
+        f1p = lambda1_p * rho_prime_ll +
+              lambda3m2_half_p / c_mean_global * rho_mean_global * v1_prime_ll +
+              (lambda2p3_half_p - lambda1_p) / c_mean_global^2 * p_prime_ll
```

Comment:
```suggestion
        f1p = (lambda1_p * rho_prime_ll +
              lambda3m2_half_p / c_mean_global * rho_mean_global * v1_prime_ll +
              (lambda2p3_half_p - lambda1_p) / c_mean_global^2 * p_prime_ll)
```
Can you please add parens around all these multiline expressions to avoid accidental errors when operators are shifted between lines (here and in the other new flux function)?

Reply:
Done in b2323e8a.